cpp/arcticdb/version/test/test_version_store.cpp

```diff
@@ -194,7 +194,7 @@ TEST_F(VersionStoreTest, SortMerge) {
         test_store_->append_incomplete_frame(symbol, std::move(frame.input_frame_));
     }
 
-    test_store_->sort_merge_internal(symbol, std::nullopt, true, false, false, false);
+    test_store_->sort_merge_internal(symbol, std::nullopt, true, false, false, false, false);
```

Comment:
5 bools in a row isn't a good API (nor was 4 before your change!). Refactor by pulling out a struct for the options?